joy_song/about_delete.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang = "en-US">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css" />
+        <link rel="stylesheet" href="style1.css">
+        <title> About Me</title>
+    </head>
+
+    <body>
+        <nav class="navbar" style="background-color: #c1cefe" id="content">
+            <div class="container">
+                <div class="navbar-brand">
+                    <a class="navbar-item brand-text" href="about.html">
+              Joy Song
+            </a>
+                    <div class="navbar-burger burger" data-target="navMenu">
+                        <span></span>
+                        <span></span>
+                        <span></span>
+                    </div>
+                </div>
+                <div id="navMenu" class="navbar-menu">
+                    <div class="navbar-start">
+                        <a class="navbar-item" href="home.html">
+                Home
+              </a>
+                        <a class="navbar-item" href="home.html">
+                Blog
+              </a>
+                    </div>
+                </div>
+            </div>
+        </nav>
+
+        <section class="hero has-text-centered" id="content">
+            <div class="hero-body">
+            <h1><p class="title is-size-3" style="font-family: Lucida Console, Courier New, monospace; color: #624CAB">About Me</p></h1>
+            </div>
+        </section>
+
+        <section id="content">
+            <div id="container">
+                <div class="section">
+                    <h2> 
+                        Introduction 
```

Comment:
Make sure to change Tanaya's example to something for your own project. If it's not needed on your website, remove it.